core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ErrorFormating.java

```diff
@@ -0,0 +1,12 @@
+package com.novoda.noplayer.internal.exoplayer.forwarder;
+
+final class ErrorFormating {
```

Comment:
Argh 2 against 1 lol.  I don't like `Formatter` for classes that are not instances, as the class itself is not a formatter but a holder for formatting methods